### PR TITLE
All claims log events mg

### PR DESCRIPTION
**Pull request by markgreenburg**
## Description
Logs certain 'help info expanded' actions within the 526 v2 app

## Testing done
- Tested locally
- Made some unit tests

## Screenshots
N/A

## Acceptance criteria
- [x] All events specified in the ticket are pushed to `window.dataLayer`

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by bshyong — APPROVED** [review]

File: src/applications/disability-benefits/all-claims/utils.jsx [hunk 1]
```diff
@@ -667,3 +668,20 @@ export const directToCorrectForm = ({
...
+ * Pushes an event to the Analytics dataLayer if the event doesn't already
+ * exist there.
+ * @param {object} event this will get pushed to `dataLayer`
+ * @param {string} key the property in the event object to use when looking for
+ *                     existing matches in the dataLayer
+ */
+export const recordEventOnce = (event, key) => {
+  const alreadyRecorded =
+    window.dataLayer &&
+    !!window.dataLayer.find(item => item[key] === event[key]);
```
Comment (cvalarida) [inline src/applications/disability-benefits/all-claims/utils.jsx]: Probably worth putting in the comment that this function assumes `event` will contain only a single property. Pushing multiple properties into the `dataLayer` [is allowed](https://developers.google.com/tag-manager/devguide#multipush).

File: src/applications/disability-benefits/all-claims/constants.js [hunk 2]
```diff
@@ -244,3 +244,23 @@ export const ATTACHMENT_KEYS = [
...
+      'Disability - Form 4142 - Private Medical Records: Read the full text',
+  },
+  openedPrivateChoiceHelp: {
+    event: 'form-help-text-clicked',
+    'help-text-label':
+      'Disability - Form 4142 - Private Medical Records: Which should I choose',
+  },
+  openedLimitedConsentHelp: {
+    event: 'form-help-text-clicked',
+    'help-text-label':
```
Comment (cvalarida) [inline src/applications/disability-benefits/all-claims/constants.js]: What do you think of rewriting these:
```js
{
  event: 'whatever-goes-here',
  [EVENT_COMPARISON_KEY]: 'Whatever goes here'
}
```

File: src/applications/disability-benefits/all-claims/constants.js [hunk 3]
```diff
@@ -244,3 +244,23 @@ export const ATTACHMENT_KEYS = [
...
+      'Disability - Form 4142 - Private Medical Records Release: What does this mean',
+  },
+};
+
+export const EVENT_COMPARISON_KEY = 'help-text-label';
```
Comment (cvalarida) [inline src/applications/disability-benefits/all-claims/constants.js]: What do you think of `HELP_TEXT_CLICKED_EVENT`?

**Review by cvalarida — APPROVED** [review]
I suggested a couple renames and a comment, but it looks otherwise good to go.

**Pull request merged**